### PR TITLE
Add guided experience summary page

### LINKED ISSUE #1485: Create a stub page for the guided experience summary
Doesn't need to do anything, just render the `GuidedExperience` component and have some tests.

### PR DESCRIPTION
**Pull request by SupeDeDupe**
Closes #1485

Created a page for guided experience summary.

### CODE REVIEW

**Comment by SupeDeDupe** [comment]
Anyway, I don't really know how to create a test file from scratch.

**Comment by SupeDeDupe** [comment]
Closes #1485

File: pages/guided_summary.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        t={t}
+        hideNoscript={false}
+        showRefreshCache={false}
+        title={t("titles.ge_summary")}
+      >
+        <GuidedExperience
+          id={"summary"}
+          stepNumber={6}
+          nextSection={"benefits-directory"}
+          /*prevSection={this.getPrevSection(
```
Comment (smcmurtry) [inline pages/guided_summary.js]: 🔥

File: pages/guided_summary.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                        displayable_sections,
+                        dynamicStepNumber
+                    )}*/
+          //setSection={this.setSection}
```
Comment (smcmurtry) [inline pages/guided_summary.js]: 🔥

File: __tests__/pages/guided_summary_test.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  });
+
+  it("passes axe tests", async () => {
+    let html = mount(<GuidedSummary {...props} />).html();
+    expect(await axe(html)).toHaveNoViolations();
+  });
+
+  it("renders page title", async () => {
+    let text = mount(<GuidedSummary {...props} />).text();
+    //console.log(text);
```
Comment (smcmurtry) [inline __tests__/pages/guided_summary_test.js]: 🔥

**Review by smcmurtry — APPROVED** [review]
Looks good! Just delete your comments before merging.

**Pull request merged**